### PR TITLE
Document query is a plugin. 

### LINKED ISSUE #1441: Query help page should mention it's a plugin
Hi,

Would it be possible to add for `query`-* page (main, web, json, xml) that it's a plugin  (nu_plugin_query) ?
It would be easier to find for beginner.

If that's OK, I can contribute to the documentation. Thanks

### PR DESCRIPTION
**Pull request by apraga**
Fixes #1441 .

### CODE REVIEW

File: make_docs.nu [hunk 1]
```diff
@@ -254,7 +254,7 @@ $"($example.description)
         $'::: warning(char nl)Dataframe commands were not shipped in the official binaries by default, you have to build it with `--features=dataframe` flag(char nl):::(char nl)(char nl)'
     } else { '' }
 
-    let plugins = if $command.name in ['from ini', 'from ics', 'from eml', 'from vcf'] {
+    let plugins = if $command.name in ['from ini', 'from ics', 'from eml', 'from vcf', 'query', 'query xml', 'query json', 'query web'] {
```
Comment (sholderbach) [inline make_docs.nu:257]: Thanks for fixing this!
The message below points to `nu_plugin_formats`. Could we make the message specific to `nu_plugin_query` as well?

Comment (apraga) [reply make_docs.nu]: Sorry about that ! It should be fixed in the new commit. By the way, I could not generate the documentation using `make_docs.nu` directly so I'm not sure it compiles.

File: make_docs.nu [hunk 2]
```diff
@@ -255,7 +255,10 @@ $"($example.description)
     } else { '' }
 
     let plugins = if $command.name in ['from ini', 'from ics', 'from eml', 'from vcf'] {
-        $"::: warning(char nl)Command `($command.name)` resides in [plugin]\(/book/plugins.html) [`nu_plugin_formats`]\(https://crates.io/crates/nu_plugin_formats). To use this command, you must install/compile and register nu_plugin_formats(char nl):::(char nl)(char nl)"
+        $"::: warning(char nl)Command `($command.name)` resides in [plugin]\(/booelk/plugins.html) [`nu_plugin_formats`]\(https://crates.io/crates/nu_plugin_formats). To use this command, you must install/compile and register nu_plugin_formats(char nl):::(char nl)(char nl)"
```
Comment (fdncred) [inline make_docs.nu:258]:
```suggestion
        $"::: warning(char nl)Command `($command.name)` resides in [plugin]\(/book/plugins.html) [`nu_plugin_formats`]\(https://crates.io/crates/nu_plugin_formats). To use this command, you must install/compile and register nu_plugin_formats(char nl):::(char nl)(char nl)"
```

File: make_docs.nu [hunk 3]
```diff
@@ -255,7 +255,10 @@ $"($example.description)
...
+    }
+    else if $command.name in ['query', 'query xml', 'query json', 'query web'] {
+        $"::: warning(char nl)Command `($command.name)` resides in [plugin]\(/booelk/plugins.html) [`nu_plugin_query`]\(https://crates.io/crates/nu_plugin_query). To use this command, you must install/compile and register nu_plugin_query(char nl):::(char nl)(char nl)"
```
Comment (fdncred) [inline make_docs.nu:261]:
```suggestion
        $"::: warning(char nl)Command `($command.name)` resides in [plugin]\(/book/plugins.html) [`nu_plugin_query`]\(https://crates.io/crates/nu_plugin_query). To use this command, you must install/compile and register nu_plugin_query(char nl):::(char nl)(char nl)"
```

File: make_docs.nu [hunk 4]
```diff
@@ -255,7 +255,10 @@ $"($example.description)
     } else { '' }
 
     let plugins = if $command.name in ['from ini', 'from ics', 'from eml', 'from vcf'] {
-        $"::: warning(char nl)Command `($command.name)` resides in [plugin]\(/book/plugins.html) [`nu_plugin_formats`]\(https://crates.io/crates/nu_plugin_formats). To use this command, you must install/compile and register nu_plugin_formats(char nl):::(char nl)(char nl)"
+        $"::: warning(char nl)Command `($command.name)` resides in [plugin]\(/booelk/plugins.html) [`nu_plugin_formats`]\(https://crates.io/crates/nu_plugin_formats). To use this command, you must install/compile and register nu_plugin_formats(char nl):::(char nl)(char nl)"
+    }
+    else if $command.name in ['query', 'query xml', 'query json', 'query web'] {
```
Comment (fdncred) [inline make_docs.nu:260]: this else statement needs to be on the line above like `} else if`

**Comment by fdncred** [comment]
if you make the changes I suggested then do `nu make_docs.nu` the script should work.

**Review by fdncred — CHANGES_REQUESTED** [review]
if you make the changes that I suggested then do `nu make_docs.nu` the script should work.

**Comment by apraga** [comment]
Thanks @fdncred for the follow-up ! Can't believe I introduced a typo... The scripts still fails (and has before my change) with a super-long error message that ends with a issue about `sort-by category`.

**Comment by fdncred** [comment]
You may need to use 0.94.3. That's what version I used to test it.

**Comment by apraga** [comment]
Even with 0.94.3, I still have the same very long error messages. I'm not sure this PR is theplace to discuss debug issues but it comes from `    let commands = (
        scope commands
        | join (command-names) name
        | sort-by category
    )
`.

**Comment by fdncred** [comment]
I've ran it twice without issues.

**Pull request merged**